datafusion/sqllogictest/bin/sqllogictests.rs

```diff
@@ -184,7 +186,11 @@ async fn run_tests() -> Result<()> {
         .collect()
         .await;
 
-    m.println(format!("Completed in {}", HumanDuration(start.elapsed())))?;
+    m.println(format!(
+        "Completed {} tests in {}",
```

Comment:
let's be explicit whether we're printing number of test files or number of test queries executed.

Reply:
```suggestion
        "Completed {} test files in {}",
```